pom.xml

```diff
@@ -40,14 +40,20 @@
   <packaging>jar</packaging>
   <url>https://github.com/nitnelave/CreeperHeal</url>
   <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.1</version>
         <configuration>
-          <source>1.6</source>
```

Comment:
There's no reason to drop Java 6 support right now, since other servers might not be all the way up to date. We can code in Java 6, unless there's a compelling reason to use Java 8.

Reply:
There's a commit for Java 8 after the 1.13 commit. 1.12+ use Java 8. Unless you opt to use some sort of [Material replacement](https://www.spigotmc.org/threads/1-8-to-1-13-itemstack-material-version-support.329630/) instead of actually updating to 1.13, there's no point in not using the features available in Java 8.

Reply:
Going to mark all the other Java 8 notations as resolved, because they all depend on this decision.

Reply:
Let's do an update to 1.12 and fix all the bugs, and then we'll do a breaking change one.

Reply:
And for 1.12 your can keep Java 6.

Reply:
1.12 itself requires Java 8

Reply:
Oh, okay. I'd still like to do the Java 8 switch in another PR, for clarity.